sw/airborne/modules/e_identification_fr/e_identification_fr.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (C) Fabien Bonneval
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/e_identification_fr/e_identification_fr.c"
+ * @author Fabien Bonneval
+ * Format and send via UART tracking data according to French requirements.
+ */
+
+#include "modules/e_identification_fr/e_identification_fr.h"
+#include "state.h"
+#include "mcu_periph/uart.h"
+#include "subsystems/gps.h"
+#include "generated/airframe.h"
+#include "generated/flight_plan.h"
+
+#if defined(ROTORCRAFT_FIRMWARE)
+#include "subsystems/navigation/waypoints.h"
+#endif
+
+#define MAX_BUF_LEN 74
+
+#define LEN_ID_FR 30
+#define LEN_ID_SERIAL 24
+
+struct uart_periph *dev;
```

Comment:
can be static, and should be static with such a generic name. Could also be initialised directly here.

Reply:
ok